Contributing.md

```diff
@@ -0,0 +1,141 @@
+
```

Comment:
Please add a heading here like 'Contribution Guide - WTF'

Reply:
@realslimshanky  I apologise for the delay.I have made the required changes